src/main/java/net/william278/velocitab/config/Settings.java

```diff
@@ -24,6 +24,8 @@ public class Settings {
     private String format = "&7[%server%] &f%prefix%%username%";
     @YamlKey("excluded_servers")
     private ArrayList<String> excludedServers = new ArrayList<>();
+    @YamlKey(("Update_Rate"))
```

Comment:
```suggestion
    @YamlKey("update_rate")
```
nit: capitalisation/style on this

Reply:
fixed